updating.md

```diff
@@ -21,3 +21,50 @@ $ bin/update-versions --hhvm-tag=HHVM-a.b.c --hsl-tag=vx.y.z
 ```
 
 See `bin/update-versions --help` for more options.
+
+# Updating to a new version of HHVM and the HSL from a Codespace
+
+First, ensure that your Codespace has a large enough machine type to perform 
+these upgrade operations. At minimum, a 16-core, 32GB machine is usually necessary.
+
+As usual, run `update-versions` to update the targetted HHVM version. 
```

Comment:
"as usual" kinda implies to me that the user ought to know this already, which seems a little discouraging.